src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -584,14 +584,6 @@ static inline int MPIDI_NM_mpi_put(const void *origin_addr,
         goto fn_exit;
     }
 
-    MPIDI_CH4U_EPOCH_CHECK_SYNC(win, mpi_errno, goto fn_fail);
```

Comment:
I guess I misunderstood. Do you mean only the PROC_NULL check should be in CH4 ? I was also a little confused how would a netmod-specific sync event trigger the epoch status change, while the status checking is in a different layer.